axelrod/tests/strategies/test_finite_state_machines.py

```diff
@@ -512,6 +512,72 @@ def test_strategy(self):
         self.transitions_test(state_and_actions)
 
 
+class TestUsuallyCooperates(TestFSMPlayer):
+    name = "UsuallyCooperates"
+    player = axelrod.UsuallyCooperates
+    expected_classifier = {
+        "memory_depth": 2,
+        "stochastic": False,
+        "makes_use_of": set(),
+        "long_run_time": False,
+        "inspects_source": False,
+        "manipulates_source": False,
+        "manipulates_state": False,
+    }
+    """
+    transitions = (
+            (1, C, 1, C),
+            (1, D, 2, C),
+            (2, C, 1, D),
+            (2, D, 1, C)
+        )
+    """
+
+    def test_strategy(self):
+        # Never leaves state 1 if C
+        state_and_actions = [(1, C)] * 10
+        self.transitions_test(state_and_actions)
+        # Visits state 2, but then comes back
```

Comment:
Can you indicate in the comments that this is testing the "except for a D following a C" part? (and below)

Reply:
I added a comment.